packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -103,19 +103,19 @@ const multiChainFetchList: { parentChain: ChainId; chain: ChainId }[] = [
     parentChain: ChainId.Goerli,
     chain: ChainId.ArbitrumGoerli
   },
-  {
-    parentChain: ChainId.Sepolia,
-    chain: ChainId.ArbitrumSepolia
-  },
+  // {
```

Comment:
Commenting out for now so you can test stuff. Sepolia throwing even in prod, needs investigation